code/game/objects/items/trash.dm

```diff
@@ -7,6 +7,18 @@
 	w_class = WEIGHT_CLASS_TINY
 	resistance_flags = FLAMMABLE
 
+/obj/item/trash/Initialize(mapload)
+	var/area/A = get_area(src)
+	if(is_station_level(z) && !istype(A, /area/maintenance))
+		SSblackbox.record_feedback("tally", "station_mess_created", 1, name)
+	return ..()
+
+/obj/item/trash/Destroy()
+	var/area/A = get_area(src)
+	if(is_station_level(initial(z)) && !istype(A, /area/maintenance))

```

Comment:
Actually I think this won't work as i.e. trash from eating food gets created in a mob's hands, where its z would be 0.

Reply:
get the turf then check the turf's z?

Reply:
you can prob just check `blob_allowed` on the area and if there's any non-station areas they should have it set to off.

Reply:
@jordie0608 Is there a more sensible way of doing this instead of adding code to the initalise and destroy of every trash item? For example, having a list of trash items that can be updated on the fly and just adding code in /obj/item/Initialize() and /obj/item/Destroy() to check if the item created is one of them?

Reply:
why would you not want certain trash items not counted in the "trashed made/clean" stat?

Reply:
Because if I'm adding this stat to see if the janitor can keep on top of mess it doesn't make sense to me to count trash in maintenance.

Reply:
I'm not understanding why you wouldn't count trash in maintenance. Janitor should be keeping that clean too since people are expected to traverse it.

Reply:
That is definitely never happening. But I'll remove the check for maint.